README.md

```diff
@@ -25,7 +25,9 @@ uv pip install -r pyproject.toml --all-extras
 uv lock
 ```
 
-Install src package with `uv pip install -e .`
+Install src package locally with `uv pip install -e .`
+
+Install src package on cluster in notebook with `pip install dbfs:/Volumes/main/default/file_exchange/nico/power_consumption-0.0.1-py3-none-any.whl`
```

Comment:
it does not install it in the cluster that way, just in your notebook env.

Reply:
@mvechtomova I know. But when I run it with databricks connect I need it locally and not in my cluster right? Because the code gets executed locally except for the spark code.